docs/source/scylla_cluster_autoscaler_crd.md

```diff
@@ -0,0 +1,104 @@
+      racks:
+      - name: us-east-1a
+        rules:
+        - name: cpu utilization horizontal up
+          priority: 1
+          expression: 'avg(scylla_reactor_utilization{scylla_cluster="example-cluster", scylla_datacenter="us-east-1", scylla_rack="us-east-1a"}) > bool 70'
+          mode: Horizontal
+          for: 10m
+          step: 30s
+          factor: 2
+        - name: cpu utilization horizontal down
+          priority: 1
+          expression: 'avg(scylla_reactor_utilization{scylla_cluster="example-cluster", scylla_datacenter="us-east-1", scylla_rack="us-east-1a"}) < bool 10'
+          mode: Horizontal
+          for: 10m
+          step: 30s
+          factor: 0.5
+        memberPolicy:
+          minAllowed: 1
+          maxAllowed: 5
+        resourcePolicy:
+          minAllowedCpu: 1
+          maxAllowedCpu: 2
+          controlledValues: Requests
+status:
+  lastApplied: 2021-04-14T11:54:17Z
+  lastUpdated: 2021-04-14T16:43:22Z
+  updateStatus: Ok
+  recommendations:
+    datacenterRecommendations:
+      - name: us-east-1
+        rackRecommendations:
+          - name: us-east-1a
+            members: 2
+            resources:
+              limits:
+                cpu:     2
+                memory:  30Gi
+              requests:
+                cpu:     2
+                memory:  30Gi
+```
+
+## Autoscaler Settings
+
+* `targetRef`: description if the ScyllaCluster object the autoscaling configuration is referring to. Comprising of `name` and `namespace`.
+  * `namespace`: namespace of ScyllaCluster
+  * `name`: name of ScyllaCluster
+
+* `updatePolicy`: Optional field. Rules and limitations of how the target is meant to be updated.
+  * `updateMode`: Optional field. Can be set to either "Off" or "Auto" (default "Auto"). Recommendations are being provided and saved in either of these cases, however, they are only ever applied in the latter.
+  * `recommendationExpirationTime`: Optional field. How long the recommendations stay valid.
+  * `updateCooldown`: Optional field. Length of a period after updating ScyllaCluster, during which no other recommendations should be applied.
+
+* `scalingPolicy`: Optional field. Rules and limitations of how specific datacenters and rack (identified by `name`) are meant to be scaled.
+  * `rules`: descriptions of boolean queries and the actions to be invoked, were their evaluated values true. A simple query is only tested at the time of evaluation. A ranged query, on the other hand, is tested against a specified time range with a predetermined frequency. It only evaluates to true if the condition has been met at all points in the time series. A single rule is composed of the following:
+    * `name`: unique name of the rule.
+    * `priority`: importance of a rule (minimum value is 0). One with the lowest priority is chosen over the others. For triggered rules with equal priority, their top to bottom order decides.
+    * `expression`: boolean query to the monitoring service.
```

Comment:
Paste a link do Promehetus query language documentation

Reply:
Ok. Changed to
`* `rules`: descriptions of boolean queries (currently [PromQL](https://prometheus.io/docs/prometheus/latest/querying/basics) format is supported) ...`